core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.html

```diff
@@ -117,31 +117,47 @@
         <div class="e2e-test-add-chapter-outline">
           <div class="story-node-header oppia-mobile-collapsible-card-header" (click)="toggleChapterOutline()">
             <span>Chapter Outline</span>
-            <i class="fa fa-caret-down"
-               *ngIf="!chapterOutlineIsShown"
-               aria-hidden="true">
-            </i>
-            <i class="fa fa-caret-up"
-               *ngIf="chapterOutlineIsShown"
-               aria-hidden="true">
-            </i>
           </div>
-          <div class="chapter-card-inputs oppia-mobile-collapsible-card-content" *ngIf="chapterOutlineIsShown">
+          <br>
+          <div class="outline-Finalized-container" *ngIf="oldOutline">
             Outline Finalized
             <i *ngIf="!outlineIsFinalized" (click)="finalizeOutline()" class="far fa-square md-18 e2e-test-finalize-outline" aria-label="Finalize outline"></i>
             <i *ngIf="outlineIsFinalized" (click)="unfinalizeOutline()" class="fas fa-check-square md-18" aria-label="Unfinalize outline"></i>
-            <div tabindex="-1" (blur)="toggleChapterOutlineButtons()" (click)="toggleChapterOutlineButtons()">
+          </div>
+          <div *ngIf="!chapterOutlineIsShown">
+            <div class="outline-content-container" *ngIf="!oldOutlineToReadOnly">
```

Comment:
oldOutlineToReadOnly is not clear. Maybe you mean !outlineEditViewIsShown or similar? (Conceptually, there should only be one outline item, and it's either editable or readonly -- oldOutline refers to a previous version of that outline.)

Reply:
I replaced `oldOutlineToReadOnly` with `outlineEditViewIsShown`.
![fix_html_ts_1](https://github.com/oppia/oppia/assets/140389350/fd96cb07-0a44-46fb-b594-98da2aa6f3ac)